Sources/CodeGenerator/Stages/GenerationStage/SourceCode.swift

```diff
@@ -8,13 +8,22 @@
 /// This model keeps generated code, name and location of file to create with this code
 public struct SourceCode {
 
-    public init(code: String, fileName: String, destinationPath: String) {
+    public init(
+        code: String,
+        fileName: String,
+        destinationPath: String,
+        apiDefinitionFileRef: String
+    ) {
         self.code = code
         self.fileName = fileName
         self.destinationPath = destinationPath
+        self.apiDefinitionFileRef = apiDefinitionFileRef
     }
 
     public let code: String
     public let fileName: String
     public let destinationPath: String
+    public let apiDefinitionFileRef: String
+    /// Suffix for files which require package separation for generation
+    public static let separatedFilesSuffix = "models.yaml"
```

Comment:
Why doest it have default value?

Reply:
Here is the static variable, because we separate models only